sandboxes/staticanalysis/staticanalyze.go

```diff
@@ -220,7 +225,7 @@ func run() (err error) {
 func main() {
 	err := run()
```

Comment:
self-nit: can we make this `if err := run(); err != nil {  ...` please?

I actually did this locally but hadn't pushed the change yet.

Reply:
Done